dcicutils/diff_utils.py

```diff
@@ -140,7 +145,7 @@ def patch_diffs(self, item):
         """
         Returns a list of keys for would-be-affected properties if item were a patch request for a piece of JSON.
         """
-        result = self.diffs({}, item, _omit_subscripts=True)
+        result = self.diffs({}, item, _omit_subscripts=True, _omit_empty_containers=False)
```

Comment:
I'm fine with defaulting to the desired behavior, but this should probably be an argument to this function as well.

Reply:
I'll take that up another time. I'm going to merge this as-is. The interim test function says tests pass.